### PR TITLE
Buyer broker pages request tdd

### PR DESCRIPTION
**Pull request by Eileenandrea**
Work on buyer and broker pages  (stock market, transaction, search Stock, and my_portfolio), Here is what's inside this PR.
-[x] html pages (stock market, transaction, search transaction, and my _portfolio).
-[x] request test for the pages above.
-[x] top navbar and side bars with pages links
-[x] ability file that specify user authorization

sidenav bar
Side Nav Bar
top nav bar  home page user is not signed -in
top nav bar if not sign in

top nav bar  home page user is signed -in
top Nav bar if sign in and page is at root route

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -1 +1,42 @@
...
+.icon {
+    font-size: 1.4rem !important;
+    margin-right: 1rem; 
+}
+.sidebar.active{
+    width: 280px !important;
+}
+.sidebar-active{
+    margin-left: 280px !important;
+}
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/custom.css.scss]: add new line

File: app/views/layouts/_side_navigation.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </a>
+      <ul class="dropdown-menu text-small shadow" aria-labelledby="dropdownUser2">
+        <li><a class="dropdown-item" href="#">New project...</a></li>
+        <li><a class="dropdown-item" href="#">Settings</a></li>
+        <li><a class="dropdown-item" href="#">Profile</a></li>
+        <li><hr class="dropdown-divider"></li>
+        <li><%= link_to('Sign out', destroy_user_session_path, method: :delete) %> </li>
+      </ul>
+    </div>
+  </div>
```
Comment (analyn-cajocson) [inline app/views/layouts/_side_navigation.html.erb]: add new line

File: app/views/layouts/_top_navigation.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+         <li class="nav-item">
+          <%=link_to new_user_registration_path,{ :class => 'nav-link link-dark'}   do%>
+            <i class="fas fa-user-plus icon"></i>Sign_up
+          <%end%>   
+         </li>
+        <%end%>
+      </ul>
+    </div>
+  </div>
+</nav>
```
Comment (analyn-cajocson) [inline app/views/layouts/_top_navigation.html.erb]: add new line

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/custom.css.scss (hunk 1 above)
Comment (Eileenandrea) [reply app/assets/stylesheets/custom.css.scss]: done po

File: app/views/layouts/_side_navigation.html.erb (hunk 2 above)
Comment (Eileenandrea) [reply app/views/layouts/_side_navigation.html.erb]: done po

File: app/views/layouts/_top_navigation.html.erb (hunk 3 above)
Comment (Eileenandrea) [reply app/views/layouts/_top_navigation.html.erb]: done po

**Review by analyn-cajocson — APPROVED** [review]

**Review by santosral — APPROVED** [review]

**Pull request merged**